### ISSUE TITLE
Slider readme example 2 incorrectly shows tooltip with data-slot

### ISSUE THREAD

**Issue by abumami** [wix/stylable-components#441]
Need to change api code example 2 to enable ::tooltip by setting the displayTooltip prop to true.

**Comment by maksymc** [comment]
Yeah, examples will be updated soon. For now you can delete this sample. Anyways it is an imaginary sample that we did before the component was written. 
This week I am going to update every spec (for Kyiv components) with relevant code samples.
